sys/include/net/fib/table.h

```diff
@@ -45,6 +45,32 @@ typedef struct fib_entry_t {
     struct universal_address_container_t *next_hop;
 } fib_entry_t;
 
+/**
+* @brief Container descriptor for a FIB source route entry
+*/
+typedef struct fib_sr_entry_t {
+    /** Pointer to the shared generic address */
```

Comment:
Doxy usually recognizes this as comment for the following definition.
I think `/**< ...*/` or `///<` is only needed if you use it in the same line.